x-pack/plugins/cases/public/components/add_comment/index.tsx

```diff
@@ -73,9 +73,13 @@ export const AddComment = React.memo(
     ) => {
       const editorRef = useRef<EuiMarkdownEditorRef>(null);
       const [focusOnContext, setFocusOnContext] = useState(false);
-      const { permissions, owner, appId } = useCasesContext();
+      const { permissions, owner } = useCasesContext();
       const { isLoading, mutate: createAttachments } = useCreateAttachments();
-      const draftStorageKey = getMarkdownEditorStorageKey(appId, caseId, id);
+      const draftStorageKey = getMarkdownEditorStorageKey({
+        appId: owner[0],
```

Comment:
why not `appId`?

Reply:
Same as https://github.com/elastic/kibana/pull/177798#discussion_r1517691882